convert-to-declarative-plugin/src/main/java/io/jenkins/plugins/todeclarative/actions/ToDeclarativeAction.java

```diff
@@ -88,7 +90,6 @@ public String getDisplayName()
     @Override
     public String getUrlName()
     {
-        this.job.checkPermission( Permission.CONFIGURE );
```

Comment:
Not what you want, this makes it so that users who don't have `Job/Configure` permission can't even see the Job page at all, because they get redirected to the login page or shown an error.

We just need to block them from actually running `doConvert` and from seeing the sidebar link.